src/config.ts

```diff
@@ -0,0 +1,148 @@
+/* eslint-disable @typescript-eslint/restrict-template-expressions */
+import path from 'path'
+import url from 'url'
+import { inspect } from 'util'
+// Load all the ENV variables from `.env`, then `.env.local`, into process.env
+import dotenv from 'dotenv-flow'
+dotenv.config()
+
+/**
+ * Database
+ */
+const db: any = {
```

Comment:
consider removing `any` to allow the compiler to warn a developer if a config member changes or get renamed in this file and he forgets to fix it in the file that consumes the config.

it will also get rid of most of the `config.db.something as string`

Reply:
Removed `any` from config.